src/components/CategoriesList/index.js

```diff
@@ -1,19 +1,27 @@
 import { categories } from '../../data/categories'
-import Link from 'next/link'
 import Image from 'next/image'
+import StyledLink from '../StyledLink'
+import { StyledList } from './CategoriesList.styled'
+import BackToLink from '../BackToLink'
+import { StyledTitle } from '../ProductDetails/ProductDetails.styled'
 
 function CategoriesList() {
   return (
-      <ul>
+    <>
+      <BackToLink href="/" alt="go to Homepage">
+        Home
+      </BackToLink>
+      <StyledList>
```

Comment:
Because you've set the `list-style-type`to `none`:
```suggestion
      <StyledList role="list">
```
[See mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns)